SConstruct

```diff
@@ -2141,14 +2148,14 @@ if not GetOption('help'):
         subdirs += ['examples']
 
     if 'tests' in COMMAND_LINE_TARGETS:
-        if env['NOSETESTS'] == '':
-            nosetests = which('nosetests')
-            if nosetests is None:
-                env['NOSETESTS'] = None
+        if env['PYTEST'] == '':
+            pytest = which('pytest')
+            if pytest is None:
+                env['PYTEST'] = None
             else:
-                env['NOSETESTS'] = nosetests
-        if env['NOSETESTS']:
-            GetNosetestsVersion(env)
+                env['PYTEST'] = pytest
+            if env['PYTEST']:
+                GetPytestVersion(env)
```

Comment:
Looks like this if block is wrongly indented. It should be at the same level of indentation as the above `if env['PYTEST'] == '':`.